<launch>
  <node name="sbg_ellipse" pkg="sbg_driver" type="ellipse" output="screen">
  	<rosparam command="load" file="$(find roboy_ad)/imu/ellipse_A_roboy.yaml" />
  </node>
</launch>
